src/utils/rgbToHSL.test.ts

```diff
@@ -0,0 +1,9 @@
+import rgbToHSL from './rgbToHSL';
+
+describe('rgbToHSL', () => {
+  test('should convert rgb to hsl correctly', () => {
+    const rgbArray = [230, 102, 34];
+    const hsl = rgbToHSL(rgbArray[0], rgbArray[1], rgbArray[2]);
+    expect(hsl).toStrictEqual([21, 80, 52]);
+  });
+});
```

Comment:
As I said in the last comment, I don't think that this is good enough testing of this conversion. I think that is good to have a good external solution to do that.